packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts

```diff
@@ -17,7 +17,8 @@ export function generateTasksJson(
   const comment = `
   // This file is automatically generated by Teams Toolkit.
   // The teamsfx tasks defined in this file require Teams Toolkit version >= 4.0.7.
-  // See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task.
+  // See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task,
```

Comment:
See https://aka.ms/teamsfx-debug-tasks for details on how to customize each task and how to integrate with existing Teams Toolkit projects.

Reply:
Done